00-dev-environment/README.md

```diff
@@ -134,27 +134,27 @@ Which AWS environment variable cannot be set in order to run the
 `aws sts get-session-token` command?
 
 > It would seem the answer is `AWS_ACCESS_KEY_ID`, based on...
-> ```sh
+> ```txt
 > $ ./awstool _test_get_session_token_with_env_vars
 > Got fresh credentials using MFA OTP.
 > Waiting 16 seconds for new OTP.
 > 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0
 > Removing: AWS_ACCESS_KEY_ID, Leaving: AWS_EXPIRATION AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
-> {
->   "AccessKeyId": "...QWVFP533"
-> }
+> { "AccessKeyId": "...QWVFP533" }
 > ---- 25 24 23 22 21 20 19 18 17 16 15 14 13 12 11 10 9 8 7 6 5 4 3 2 1 0 ----
```

Comment:
What's up with your line breaks here? I haven't seen people do it this way and I'm just curious about it, if there is a rhyme or reason for the numbers as they chagne.